README.md

```diff
@@ -357,14 +352,23 @@ Alternatively, to discard multiple files in your working copy, list them all.
 $ git checkout myFirstFile mySecondFile
 ```
 
-### I want to discard all of my local changes
+<a href="i-want-to-discard-only-my-unstaged-local-changes"></a>
+### I want to discard only my unstaged local changes
 
-When you want to get rid of all of your local uncommitted changes
+When you want to get rid of all of your unstaged local uncommitted changes
 
 ```sh
 $ git checkout .
 ```
 
+### I want to discard all of my untracked files
+
+When you want to get rid of all of your untracked files
+
+```sh
+$ git clean -f
```

Comment:
Why not `-d` and `-x`, too?

Reply:
I have made the other two changes. I will send out a new commit for -x and -d flags during checkout

Reply:
I have made the other two changes. I will send out a new commit for -x and -d flags during checkout

Reply:
What do you mean by checkout?

Reply:
(checkout .) gets the latest files from remote and disregards local unstaged changes. This is a common practice to discard local changes to restore working directory to original state.

Reply:
If you're asking about checkout -f, then no, I did not add that and just left it there as it was already there.